.gitignore

```diff
@@ -1,7 +1,7 @@
-*/*.log
-/*.log
+*.log
 /.vagrant/*
 */.idea/*
 *.box
 *.pyc
 _pycache_/
+*.sql
```

Comment:
Generally sql files contain queries that would probably be needed for the app.
Do you want to ignore all .sql files or only one specific file (db.sql)

Reply:
I'll change that to ignore only db.sql

Reply:
I squashed some commits so this file is no longer reachable through this commit.
So I'm resolving this conversation.